docs/windows.md

```diff
@@ -0,0 +1,187 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports to run on Windows worker Node, and setups an overlay network
+which uses VXLAN tunneling to route packets between nodes.
+
+One HNS Network with [Transparent mode](https://docs.microsoft.com/en-us/virtualization/windowscontainers/container-networking/network-drivers-topologies)
+is created on a Windows worker Node, on which Open vSwitch Extension is enabled.
+The HNS Network uses the network adapter which is used to join the Windows Node
```

Comment:
I find this sentence a bit unclear. I assume you mean the network adapter whose IP address was used to join the cluster.

I suggest this as a replacement: The HNS Network connects directly to the physical network through the network adapter whose IP address was used as the Node IP when the Windows Node joined the cluster.

Let me know if this is inaccurate.

Reply:
Thanks Antonin, that's better.